_components/accordions.md

```diff
@@ -80,7 +79,7 @@ title: Accordions
         Clicking anywhere on the header (not just on the icon) triggers expansion or collapse of that section.
       </li>
       <li>
-        Any interactive element within each section should be placed a minimum of 2 mm from the header to avoid accidentally triggering the wrong interaction.
+        Any interactive element within each section should be placed a minimum of 10 pixels from the header to avoid accidentally triggering the wrong interaction.
```

Comment:
So the tricky thing here is when screens have differing pixel density. I agree that the 2mm is a confusing measurement and not very useful, but it's based on the size of an average finger/thumb tip (touch size). 10px on a low resolution screen is smaller than 10 px on a high resolution screen. See http://www.lukew.com/ff/entry.asp?1085 

I wish there was an easier way to provide guidance on this. Maybe we just need to be more vague and say "The average adult fingertip is between 8-10 millimeters in diameter; make sure any interactive elements within the collapsable region are far enough from the headers so that users don't accidentally trigger a collapse."

???